src/styles/blocks/product-catalog.scss

```diff
@@ -0,0 +1,29 @@
+.product-catalog {
+  padding-block: 50px;
+  padding-inline: 40px;
+
+  display: grid;
+  grid-template-columns: repeat(1, minmax(200px, 200px));
+  grid-auto-rows: 1fr;
+  gap: 46px 48px;
+  justify-content: center;
+  justify-items: center;
+
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, minmax(200px, 200px));
```

Comment:
Here as well, you can simplify `minmax(200px, 200px)` to `200px` since the minimum and maximum values are the same, which makes `minmax` unnecessary.